unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/PublicChatChannelComponentView.cs

```diff
@@ -19,6 +19,7 @@ public class PublicChatChannelComponentView : BaseComponentView, IChannelChatWin
     public bool IsActive => gameObject.activeInHierarchy;
     public IChatHUDComponentView ChatHUD => chatView;
     public RectTransform Transform => (RectTransform) transform;
+    public string channel => nameLabel.text;
```

Comment:
you should retrieve this data from `model.name`

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/2257/commits/410c110f18ce9106cac990ce5b74417f457f161c